youtube_dl.go

```diff
@@ -257,6 +257,9 @@ func PerformGetRequest(url string) (*jsonq.JsonQuery, error) {
 		if response.StatusCode == 200 {
 			if body, err := ioutil.ReadAll(response.Body); err == nil {
 				jsonString = string(body)
+				if jsonString[0] == '[' {
```

Comment:
the soundcloud search api returns an array "[ ... ]" instead of json "{ ... }".
and this can't be parsed by apiResponse